travis-build.bash

```diff
@@ -17,7 +17,7 @@ function err() {
 function main() {
     msg "branch is ${TRAVIS_BRANCH}"
 
-    local mvn="mvn --settings .settings.xml -B -V"
+    local mvn="mvn --settings .settings.xml -B -V -DskipTests"
```

Comment:
Initially yes, but I also had a bug where it was doing a listCommits on every event issue found when it only should have it done it on commit events